internal/app.go

```diff
@@ -294,15 +314,36 @@ func (a *App) checkin(client *http.Client, crypto CryptoHandler) error {
 	if err != nil {
 		return fmt.Errorf("Unable to get: %s - %v", a.configUrl, err)
 	}
+	defer res.Body.Close()
+
 	if res.StatusCode == 200 {
+		body, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return fmt.Errorf("Unable to read new secrets: %w", err)
+		}
+
+		var config configSnapshot
+		if config.next, err = UnmarshallBuffer(crypto, body, true); err != nil {
+			return err
+		}
+		if config.prev, err = UnmarshallFile(nil, a.EncryptedConfig, false); err != nil {
+			log.Printf("Unable to load previous config version: %s", err)
```

Comment:
why don't we move this to after the `!os.IsNotExists` check so that we only log when its a real unexpected error.

Reply:
Makes sense